src/Integrations/Clearbit/ClearbitClient.php

```diff
@@ -0,0 +1,317 @@
+			'company-country' => $company['geo']['country'] ?? '',
+			'company-country-code' => $company['geo']['countryCode'] ?? '',
+			'company-lat' => $company['geo']['lat'] ?? '',
+			'company-lng' => $company['geo']['lng'] ?? '',
+			'company-logo' => $company['logo'] ?? '',
+			'company-facebook' => $company['facebook']['handle'] ?? '',
+			'company-facebook-likes' => $company['facebook']['likes'] ?? '',
+			'company-linkedin' => $company['linkedin']['handle'] ?? '',
+			'company-twitter' => $company['twitter']['handle'] ?? '',
+			'company-twitter-id' => $company['twitter']['id'] ?? '',
+			'company-twitter-bio' => $company['twitter']['bio'] ?? '',
+			'company-twitter-followers' => $company['twitter']['followers'] ?? '',
+			'company-twitter-following' => $company['twitter']['following'] ?? '',
+			'company-twitter-location' => $company['twitter']['location'] ?? '',
+			'company-twitter-site' => $company['twitter']['site'] ?? '',
+			'company-twitter-avatar' => $company['twitter']['avatar'] ?? '',
+			'company-crunchbase' => $company['crunchbase']['handle'] ?? '',
+			'company-type' => $company['type'] ?? '',
+			'company-ticker' => $company['ticker'] ?? '',
+			'company-phone' => $company['phone'] ?? '',
+			'company-alexa-us-rank' => $company['metrics']['alexaUsRank'] ?? '',
+			'company-alexa-global-rank' => $company['metrics']['alexaGlobalRank'] ?? '',
+			'company-employees' => $company['metrics']['employees'] ?? '',
+			'company-employees-range' => $company['metrics']['employeesRange'] ?? '',
+			'company-market-cap' => $company['metrics']['marketCap'] ?? '',
+			'company-raised' => $company['metrics']['raised'] ?? '',
+			'company-annual-revenue' => $company['metrics']['annualRevenue'] ?? '',
+			'company-estimated-annual-revenue' => $company['metrics']['estimatedAnnualRevenue'] ?? '',
+			'company-fiscal-year-end' => $company['metrics']['fiscalYearEnd'] ?? '',
+		];
+	}
+
+	/**
+	 * Map service messages with our own.
+	 *
+	 * @param string $msg Message got from the API.
+	 *
+	 * @return string
+	 */
+	private function getErrorMsg(string $msg): string
+	{
+		switch ($msg) {
+			case 'auth_required':
+				return 'clearbitAuthRequired';
+			default:
+				return 'submitWpError';
+		}
+	}
+
+	/**
+	 * Set headers used for fetching data.
+	 *
+	 * @return array<string, mixed>
+	 */
+	private function getHeaders(): array
+	{
+		return [
+			'Content-Type' => 'application/json; charset=utf-8',
+			'Authorization' => "Bearer {$this->getApiKey()}",
+		];
+	}
+
+	/**
+	 * Return Api Key from settings or global variable.
+	 *
+	 * @return string
+	 */
+	private function getApiKey(): string
+	{
+		$apiKey = Variables::getApiKeyClearbit();
+
+		return $apiKey ? $apiKey : $this->getOptionValue(SettingsClearbit::SETTINGS_CLEARBIT_API_KEY_KEY); // phpcs:ignore WordPress.PHP.DiscouragedPHPFunctions.obfuscation_base64_encode
+	}
+
+	/**
+	 * Return Clearbit base url.
+	 *
+	 * @return string
+	 */
+	private function getBaseUrl(): string
```

Comment:
This method is only used once in the class and is returning the URI string. I'd put this as a `private const` instead.

Reply:
i would like to have consistent client files for all integrations. In hubspot I have multiple Urls so I would like to leave this as is

Reply:
Sure, but this is a private method that is different than the other one (the other ones have arguments, this one does not). So they're not that consistent 😅 

If you see certain parts repeating across files you can always abstract it out in an abstract class like a template pattern.